src/components/addBook/BookForm.jsx

```diff
@@ -19,7 +19,11 @@ const BookForm = () => {
       author,
       category: 'Fiction',
     };
-    dispatch(addBook(newBook));
+
+    dispatch(addNewBook(newBook));
+
+    // // Fetch updated books data
+    // dispatch(fetchBooks());
```

Comment:
- [ ] Please remove commented lines of code as they make your codebase look cluttered and difficult to read 👍